CodeReview/CMakeLists.txt

```diff
@@ -20,10 +21,13 @@ add_library(CodeReview SHARED
 	src/items/VCommentedNodeStyle.h
 	src/items/VReviewComment.h
 	src/items/VReviewCommentStyle.h
+	src/orderings/Orderings.h
+	src/groupings/UseAnalysisGroupings.h
```

Comment:
So I'm not convinced about the the names of these classes and the directories they are in, but we can adjust this later, once we have more strategies.